src/node.cc

```diff
@@ -3901,9 +3906,7 @@ Environment* CreateEnvironment(Isolate* isolate,
       HandleCleanup,
       nullptr);
 
-  if (v8_is_profiling) {
-    StartProfilerIdleNotifier(env);
-  }
+  StartProfilerIdleNotifier(env);
```

Comment:
Removed this change. It can be done in v8-profiler indeed, even though it's more work (and slight code duplication).